packages/amplication-client/src/VersionControl/Commit.tsx

```diff
@@ -73,10 +73,15 @@ const Commit = ({ projectId, noChanges }: Props) => {
   };
 
   const [commit, { error, loading }] = useMutation<TData>(COMMIT_CHANGES, {
-    onError: () => {
+    onError: (error: ApolloError) => {
       setCommitRunning(false);
       setPendingChangesError(true);
       setOpenLimitationDialog(true);
+      const errorMessage = formatError(error);
+      const isLimitationError =
```

Comment:
we should use isLimitationError to decide whether to use setOpenLimitationDialog

Reply:
I don't understand this comment